cmd/agent/install_mac_os.sh

```diff
@@ -60,6 +60,11 @@ if [ -n "$DD_AGENT_MINOR_VERSION" ]; then
   fi
 fi
 
+# Cleanup tmp files used for installation
+if [ -f /tmp/install-ddagent/system-wide ]; then
```

Comment:
I agree small nitpick is that if a directory exist at said location it will make the script fail. But it is very unlikely so I'll still replace it with `rm -f` 👍